meteor-app/imports/ui/layouts/full-window/style.less

```diff
@@ -10,30 +10,25 @@
   align-items: stretch;
 
   .page__header {
-    flex: 0 0 auto;
-  }
-
-  .page__body {
-    flex: 1 1 0;
-  }
-
-  .page__footer {
-    flex: 0 0 auto;
-  }
+    .flex-row;
 
-  .page__header {
     padding: @global-layout-padding;
     margin-bottom: @global-layout-margin;
     background-color: white;
   }
 
   .page__body {
+    flex: 1 1 0;
```

Comment:
The flexbox config was split away since it is only relevant to the parent-children relationships. Puting it inside here makes the code harder to port.